### PR TITLE
[DOPS-2746] Add sonar, dojo

### PR DESCRIPTION
**Pull request by C4tWithShell**
1. [Added sonar](https://sonar.soramitsu.co.jp/dashboard?id=sora%3Asora2-common&branch=feature%2FDOPS-2746%2Fsonar_dojo)
2. [Added DefectDojo](https://defectdojo.tachi.soramitsu.co.jp/test/12379)
3. Modify clip script to generate report in json format

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -6,6 +6,9 @@ def pipeline = new org.rust.AppPipeline(steps: this,
       buildTestCmds: ['housekeeping/tests.sh'],
       cargoClippyTag: ':latest',
       cargoClippyCmds: ['housekeeping/clippy.sh'],
-      codeCoverage: false
+      codeCoverage: false,
```
Comment (wpi86) [inline Jenkinsfile:9]: Why?

**Review by wpi86 — CHANGES_REQUESTED** [review]

**Review by alexkg — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (C4tWithShell) [reply Jenkinsfile:9]: There is no coverage script and env image does not have needed toosl as well.
I will discuss it with devs

File: housekeeping/clippy.sh [hunk 2]
```diff
@@ -4,8 +4,8 @@ set -e
 if [ "$pr" = true ] && [ "$prBranch" != "master" ]; then
     printf "👷‍♂️ starting clippy \n"
     SKIP_WASM_BUILD=1 cargo clippy
-    SKIP_WASM_BUILD=1 cargo clippy --features runtime-benchmarks
+    SKIP_WASM_BUILD=1 cargo clippy --features runtime-benchmarks  --message-format=json > clippy_report.json
```
Comment (Alexey-N-Chernyshov) [inline housekeeping/clippy.sh]: '-D' was missing, it is better to add the option.

**Review by Alexey-N-Chernyshov — APPROVED** [review]

**Review by Cre-eD — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (C4tWithShell) [reply Jenkinsfile:9]: Discussed with Alexey - to add it we need to update project. He will create a task

**Review by wpi86 — APPROVED** [review]

**Review by Cre-eD — APPROVED** [review]

**Pull request merged**